svthalia/concrexit
# Make documents member-only by default

Idea from #2084


### Summary

This makes the default value for this field 'true' so that making some public is an action instead of the other way around.

### How to test
Steps to test the changes you made:
1. Go to documents admin
2. Create a new document
3. See that the default setting for members-only is true
